.github/workflows/ci.yml

```diff
@@ -68,12 +74,13 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build and Publish image into GHCR
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v4
         with:
           context: .
           file: ./Dockerfile
           push: true
-          tags: ghcr.io/dragonflydb/operator:${{ github.ref_name }}
+          tags: ghcr.io/${{ github.actor }}/operator:${{ github.ref_name }}
```

Comment:
you need to revert this line. Because our operator images are released under the `ghcr.io/dragonflydb/operator` name and it doesn't make sense to use `${{ github.actor }}` here.